engines/ultima/ultima8/kernel/object_manager.cpp

```diff
@@ -288,6 +288,22 @@ bool ObjectManager::load(Common::ReadStream *rs, uint32 version) {
 	return true;
 }
 
+void ObjectManager::saveObject(Common::WriteStream *ws, Object *obj) const {
+	const char *cname = obj->GetClassType()._className; // note: virtual
+	uint16 clen = strlen(cname);
+
+	Std::map<Common::String, ObjectLoadFunc>::iterator iter;
+	iter = _objectLoaders.find(cname);
+	if (iter == _objectLoaders.end()) {
+		perr << "Object class cannot save without registered loader: " << cname << Std::endl;
+		return;
```

Comment:
Same issue as save process without load ability

Reply:
Since this is the save process, maybe best to assert here with `error()`? Unlike load (where maybe the file is corrupt or it comes from an incompatible version), it suggests a coding error if save fails.